static/images/icons/FontAwesome/slack-hash.svg

```diff
@@ -0,0 +1,3 @@
+<svg aria-hidden="true" width="0.875em" style="-ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);" preserveAspectRatio="xMidYMid meet" viewBox="0 0 448 512">
```

Comment:
Thanks for putting this in the right spot!

Can you change the contents of this file to:
```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
  <title>Slack Logo</title>
  <path d="M446 270c-6-19-27-29-46-23l-45 16-30-90 45-16c19-6 29-26 23-45s-27-30-46-23l-46 15-15-47a36 36 0 0 0-69 23l16 47-94 31-16-47a36 36 0 0 0-69 23l16 47-45 15c-19 6-29 27-23 46 5 14 19 24 33 24 7 1 12-1 58-16l30 90-45 15c-19 6-29 27-23 46 5 14 19 24 34 24 6 1 12-1 57-16l16 47c6 16 25 29 46 22 19-6 29-26 23-46l-16-46 94-32 16 47c5 17 24 29 46 23 19-6 29-27 23-46l-16-47 45-15c19-6 29-27 23-46zm-254 48l-30-91 93-31 31 90-94 32z"/>
</svg>
```